packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -636,27 +640,22 @@ export class MgtPersonCard extends MgtTemplatedComponent {
     if (getEmailFromGraphEntity(person)) {
       email = html`
         <fluent-button class="icon"
-          @click=${() => this.emailUser()}
-          @mouseenter=${() => this.setHoveredState('email', true)}
-          @mouseleave=${() => this.setHoveredState('email', false)}>
-          <span class=${classMap({ filled: this.isHovered('email'), regular: !this.isHovered('email') })}>
-            ${getSvg(SvgIcon.SmallEmail)}
-          </span>
+          aria-label="${this.strings.emailButtonLabel} ${person.displayName}"
+          @click=${this.emailUser}>
+          ${getSvg(SvgIcon.SmallEmail)}
         </fluent-button>
       `;
     }
 
     // Chat
     let chat: TemplateResult;
     if (userPerson?.userPrincipalName) {
+      const ariaLabel = `${this.strings.chatButtonLabel} ${person.displayName}`;
```

Comment:
I really like creating the ariaLabel const here for the string interpolation and using it in the html tag template, can we use this approach on the other buttons too please? or do the interpolation inline as with the others. 
I'd just like us to be consistent with approach

Reply:
Yes, updating with const ariaLabel on the others